scripts/automation_generate.sh

```diff
@@ -1,22 +1,10 @@
 #!/bin/bash
 
 TEMP_FILE="$TMPDIR/auto_temp.json"
-# generate code
```

Comment:
Remove https://github.com/Azure/azure-sdk-for-python/blob/bea283073e3877fdbc85b7bb9521f67357000e76/tools/azure-sdk-tools/setup.py#L44C42-L44C53 after delete sdk_package.py